src/main/resources/static/data-upload.js

```diff
@@ -149,5 +155,57 @@ function handle_error() {
 }
 
 $(document).ready(function () {
-    retrieveSessions();
+    $("#data-proceed-btn").attr("disabled","disabled");
+
+    $.ajax({
+        url: "/username",
+        type: "get",
+        success: function (data) {
+            USERNAME = data;
+        }
+    });
+
+    //for testing
+    $.ajax({
+        url: "/create-session",
+        type: "post",
+        data: {
+            name:"sessions1"
+        },
+        success: function (data) {
```

Comment:
instead of defining function in-place, defining it earlier and as a variable would make it more readable.